packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -182,6 +182,15 @@ describe('validation', function() {
     });
 
 
+    describe('zeebe:property', function() {
```

Comment:
I like this approach 👍 

As you mentioned we are not consistent on this. I think in C7 we have at least some test cases for particular binding types (e.g. `camunda-in-binding`, `camunda-out-binding`), but also not super strict. In C8 we don't have something like this at all, only indirectly in some of the samples. So +1 on this approach.

We can adopt it once we touch it next time.